cmd/fixer.go

```diff
@@ -489,10 +492,95 @@ var linkedAppFixer = &cobra.Command{
 	},
 }
 
+var contentMismatch64Kfixer = &cobra.Command{
+	Use:   "content-mismatch <domain>",
+	Short: "Fix the content mismatch differences for 64K issue",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Usage()
+		}
+		domain := args[0]
+		corruptedSuffix := "-corrupted"
+
+		if !noDryRunFlag {
+			fmt.Println("This is a dry-run, no file will be altered")
+		}
+
+		c := newAdminClient()
+		res, err := c.Req(&request.Options{
+			Method: "GET",
+			Path:   "/instances/" + url.PathEscape(domain) + "/fsck",
+		})
+		if err != nil {
+			return err
+		}
+
+		inst, err := lifecycle.GetInstance(domain)
+		if err != nil {
+			return fmt.Errorf("Cannot find instance %s", domain)
+		}
+
+		var content map[string]interface{}
+		scanner := bufio.NewScanner(res.Body)
+
+		for scanner.Scan() {
+			err = json.NewDecoder(bytes.NewReader(scanner.Bytes())).Decode(&content)
+			if err != nil {
+				return err
+			}
+
+			// Filtering the 64kb mismatch issue
+			if content["type"] != "content_mismatch" {
+				continue
+			}
+			cm := content["content_mismatch"].(map[string]interface{})
+			sizeIndex := cm["size_index"].(float64)
+			sizeFile := cm["size_file"].(float64)
+
+			// SizeFile should be 64k shorter than SizeIndex
+			size := int64(64 * 1024)
+			if math.Abs(sizeIndex-sizeFile) != float64(size) {
```

Comment:
No, we don't compare with `==` or `!=` two floats. You should use some structs to parse the JSON response to force an integer type.